src/api/middleware/validators/offer.js

```diff
@@ -18,6 +18,27 @@ const {
     OFFER_EDIT_GRACE_PERIOD_HOURS,
 } = require("../../../models/constants/TimeConstants");
 
+const jobMaxDurationGreaterOrEqualThanJobMinDuration = (jobMaxDuration, { req }) => {
+    try {
+        const { jobMinDuration } = req.body;
+
+        // jobMinDuration is required if jobMaxDuration was specified
+        if (!jobMinDuration) {
+            throw new Error(ValidationReasons.JOB_MIN_DURATION_NOT_SPECIFIED);
+        }
```

Comment:
create a "mustSpecifyJobMinDurationIfJobMaxDurationSpecified" custom validator and put this logic there instead, calling it before this one, since that is another rule and it will become clearer that way

Reply:
By doing that, you can remove the trycatch wrapper, and this function will get really small :)